src/format.py

```diff
@@ -108,6 +110,27 @@ def getScreenIndex(self):
     def getPath(self):
         return self.path
 
+    def getSizeInBytes(self):
+        size = os.stat(self.path).st_size
+        return 'size: ' + str(size) + ' bytes'
```

Comment:
for extra credit credit, itd be cool if this was human readable like x kB, MB, etc